### ISSUE TITLE
Graph Convolution on drugs

### ISSUE THREAD

**Issue by jananikrish92** [deepchem/deepchem#1194]
We have only smiles data from BindingDB, we want to create graph convolution. In the example on Tox21 the dataloader takes in tasks which we do not have. Could anyone give a way to execute the graph convolution for other datasets?

**Comment by rbharath** [comment]
You should be able to do this. Try checking out our graph convolutional tutorials for examples of building graph convs from smiles.

**Comment by jananikrish92** [comment]
We have another model for that is written in tensorflow, we want to combine the output of graph conv and the other model to further train the entire system. How can we get the result of the graph conv using tensorflow (not using fit_generator)?

**Comment by rbharath** [comment]
Hmm, as I understand it, you want to get the output layer from the graph-conv so you can combine it with your custom TF model to jointly train.

@peastman Does the new callable support for `TensorGraph` enable these types of use cases?

**Comment by peastman** [comment]
It does indeed.  When working in eager mode, you can pass inputs to the model and get the corresponding outputs represented as tensors.  Then you can do more calculations based on those, and gradients should be tracked correctly through the whole process.  The code was just merged today so we don't have a tutorial on it yet, but hopefully we will soon: see #1158 and particularly https://github.com/deepchem/deepchem/pull/1158#issuecomment-377975357.
